### PR TITLE
Owl Burial Ground

### PR DESCRIPTION
**Pull request by vtange**
lots of souls here.
My rule of thumb:
Spirits actively talk and give quests.
Souls are just entities you interact lightly with to finish quests.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -5451,12 +5451,12 @@ QUEST_LV_0100_20150317_005446	{memo X}
 QUEST_LV_0100_20150317_005447	Talk to Hames
 QUEST_LV_0100_20150317_005448	You've completed the amount you bet with Hames. Talk to him.
 QUEST_LV_0100_20150317_005449	{memo X}
-QUEST_LV_0100_20150317_005450	Talk to grave manager Romas
-QUEST_LV_0100_20150317_005451	Romas is looking at you with pity. Talk to him.
-QUEST_LV_0100_20150317_005452	Pass the test of Romas
-QUEST_LV_0100_20150317_005453	Romas says he had to bury a revelator once so do not be over confident. Defeat 15 monsters nearby within 2 minutes and prove taht you are not weak.
-QUEST_LV_0100_20150317_005454	Defeated 15 monsters in 2 minutes. Talk to Romas.
-QUEST_LV_0100_20150317_005455	Defeat monsters within 2 minutes
+QUEST_LV_0100_20150317_005450	Talk to Grave Guard Romas
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Grave Guard -> Gravekeeper

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -5451,12 +5451,12 @@ QUEST_LV_0100_20150317_005446	{memo X}
...
+QUEST_LV_0100_20150317_005451	$Romas is looking at you with pity. Talk to him.
+QUEST_LV_0100_20150317_005452	$Pass Romas' Test
+QUEST_LV_0100_20150317_005453	$Romas says he had to bury a Revelator once so do not be over confident. Defeat 15 monsters nearby within 2 minutes and prove that you are not weak.
+QUEST_LV_0100_20150317_005454	$Defeated 15 monsters in 2 minutes. Talk to Romas.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Talk to Romas. -> Talk to Gravekeeper Romas.

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -5451,12 +5451,12 @@ QUEST_LV_0100_20150317_005446	{memo X}
...
 QUEST_LV_0100_20150317_005449	{memo X}
-QUEST_LV_0100_20150317_005450	Talk to grave manager Romas
-QUEST_LV_0100_20150317_005451	Romas is looking at you with pity. Talk to him.
-QUEST_LV_0100_20150317_005452	Pass the test of Romas
-QUEST_LV_0100_20150317_005453	Romas says he had to bury a revelator once so do not be over confident. Defeat 15 monsters nearby within 2 minutes and prove taht you are not weak.
-QUEST_LV_0100_20150317_005454	Defeated 15 monsters in 2 minutes. Talk to Romas.
-QUEST_LV_0100_20150317_005455	Defeat monsters within 2 minutes
+QUEST_LV_0100_20150317_005450	Talk to Grave Guard Romas
+QUEST_LV_0100_20150317_005451	$Romas is looking at you with pity. Talk to him.
+QUEST_LV_0100_20150317_005452	$Pass Romas' Test
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Pass Romas' Test -> Pass Gravekeeper Romas' Test

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -5451,12 +5451,12 @@ QUEST_LV_0100_20150317_005446	{memo X}
...
 QUEST_LV_0100_20150317_005448	You've completed the amount you bet with Hames. Talk to him.
 QUEST_LV_0100_20150317_005449	{memo X}
-QUEST_LV_0100_20150317_005450	Talk to grave manager Romas
-QUEST_LV_0100_20150317_005451	Romas is looking at you with pity. Talk to him.
-QUEST_LV_0100_20150317_005452	Pass the test of Romas
-QUEST_LV_0100_20150317_005453	Romas says he had to bury a revelator once so do not be over confident. Defeat 15 monsters nearby within 2 minutes and prove taht you are not weak.
-QUEST_LV_0100_20150317_005454	Defeated 15 monsters in 2 minutes. Talk to Romas.
-QUEST_LV_0100_20150317_005455	Defeat monsters within 2 minutes
+QUEST_LV_0100_20150317_005450	Talk to Grave Guard Romas
+QUEST_LV_0100_20150317_005451	$Romas is looking at you with pity. Talk to him.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Romas -> Gravekeeper Romas

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -5451,12 +5451,12 @@ QUEST_LV_0100_20150317_005446	{memo X}
...
+QUEST_LV_0100_20150317_005452	$Pass Romas' Test
+QUEST_LV_0100_20150317_005453	$Romas says he had to bury a Revelator once so do not be over confident. Defeat 15 monsters nearby within 2 minutes and prove that you are not weak.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Romas -> Gravekeeper Romas

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -8128,27 +8128,27 @@ QUEST_LV_0100_20150918_008123	The Goddesses disappearing, things like golems app
 QUEST_LV_0100_20150918_008124	How was it? Were there many skills that seemed useful?{nl}If you use skills effectively, they will be very useful in combat.{nl}
 QUEST_LV_0100_20150918_008125	If that's the case, you will become bait and take care of the Royal Mausoleum defense device at Sesija Parvise.{nl}I will only say this once, so you better listen carefully.
 QUEST_LV_0100_20150918_008126	No one can disturb the eternal sleep of the great king.
-QUEST_LV_0100_20150918_008127	Persuade the spirit that is heading towards Amolallul Hills{nl}and guide it to the sad owl sculpture!
+QUEST_LV_0100_20150918_008127	$Persuade the spirit that is heading towards Amolallul Hills{nl}and guide it to the Sad Owl Sculpture!
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Amolallul Hills -> Amolallul Hill

File: QUEST_LV_0200.tsv [hunk 7]
```diff
@@ -892,9 +892,9 @@ QUEST_LV_0200_20150317_000891	I'm roughly done studying this area but we need th
 QUEST_LV_0200_20150317_000892	If we get the reports, even Naktis' curse will just be a mere spell. {nl}Yup, that's right. Great work. {nl}
 QUEST_LV_0200_20150317_000893	Isn't it scary in here? {nl}Even the owl statue feels gloomy..
 QUEST_LV_0200_20150317_000894	{memo X}I've heard about sightings of souls with resentment. {nl}If it's true.. Uhh, you're not one of them, are you?
-QUEST_LV_0200_20150317_000895	Grave guard Romas
-QUEST_LV_0200_20150317_000896	Being a Revelator doesn't mean that you can roam around freely tomorrow as you do today.
-QUEST_LV_0200_20150317_000897	Every Revelator who said that they were different never really made it long.
+QUEST_LV_0200_20150317_000895	Grave Guard Romas
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:895]: Grave Guard Romas -> Gravekeeper Romas

File: QUEST_LV_0200.tsv [hunk 8]
```diff
@@ -1284,25 +1284,25 @@ QUEST_LV_0200_20150317_001283	Support of Rearguard Troops
...
-QUEST_LV_0200_20150317_001295	Why the monsters are destroying owl statue
-QUEST_LV_0200_20150317_001296	Dangerous Throneweaver (2)
-QUEST_LV_0200_20150317_001297	I will get the roots of Sakmoli
-QUEST_LV_0200_20150317_001298	Restrained Soul
-QUEST_LV_0200_20150317_001299	I will check around the Nyelasve Shore
+QUEST_LV_0200_20150317_001295	Why are the monsters destroying owl statues
+QUEST_LV_0200_20150317_001296	$Dangerous Throneweaver (2)
+QUEST_LV_0200_20150317_001297	$I will get Sakmoli Roots
+QUEST_LV_0200_20150317_001298	$Restrained Souls
+QUEST_LV_0200_20150317_001299	$I will check around Nelasve Fork
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1299]: Nelasve Fork -> Nelasve Shore

File: QUEST_LV_0200.tsv [hunk 9]
```diff
@@ -1284,25 +1284,25 @@ QUEST_LV_0200_20150317_001283	Support of Rearguard Troops
...
 QUEST_LV_0200_20150317_001300	Everything will be alright
-QUEST_LV_0200_20150317_001301	Wrong Salvation (1)
-QUEST_LV_0200_20150317_001302	I will pursuade the soul and send him there
+QUEST_LV_0200_20150317_001301	$Wrong Salvation (1)
+QUEST_LV_0200_20150317_001302	$I will persuade the souls and send them here
 QUEST_LV_0200_20150317_001303	About the missing souls
 QUEST_LV_0200_20150317_001304	{memo X}Notice/!/Pursuade the wandering soul in Nelasve Shore and guide it back to the owl!{nl}You need to talk and pursuade the soul when you meet him!#5
-QUEST_LV_0200_20150317_001305	Wrong Salvation (1)
+QUEST_LV_0200_20150317_001305	$Wrong Salvation (1)
 QUEST_LV_0200_20150317_001306	I'll go to Amolarul Hills
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1306]: Amolarul Hills -> Amolallul Hill

File: QUEST_LV_0200.tsv (hunk 8 above)
Comment (vtange) [inline QUEST_LV_0200.tsv:1299]: Ok so the map name was wrong this time

File: QUEST_LV_0200.tsv [hunk 10]
```diff
@@ -2058,49 +2058,49 @@ QUEST_LV_0200_20150317_002057	You can see Commander Danius who looks exhausted.
...
 QUEST_LV_0200_20150317_002098	{memo X}The sorrowful owl told you that the spirits that went to the statue of the Goddess for comfort are agonizing themselves so something must be wrong. Persuade the spirits that try to go to the Goddess Statue and take them to the sorrowful owl.
 QUEST_LV_0200_20150317_002099	{memo X}The sorrowful owl heard enough stories about what's happening to the spirits. Go back to the sorrowful owl and ask what happened.
 QUEST_LV_0200_20150317_002100	The sorrowful owl talked enough with the spirits. Talk to the sorrowful owl again.
-QUEST_LV_0200_20150317_002101	Find out what happened at Amolallul Hills.
-QUEST_LV_0200_20150317_002102	The spirits are claming that the statue of the Goddess at Amolallul Hills were calling them but, the sorrowful owl told you that the statue wasn't there. Go to Amollaul Hills to find out exactly what happened.
-QUEST_LV_0200_20150317_002103	Corrupteds are residing at the statue of the Goddess. Demons calling themselves as the goddesses can never happen. Go back to the sorrowful owl and tell this fact.
+QUEST_LV_0200_20150317_002101	$Find out what happened at Amolallul Hills.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:2101]: Amolallul Hills -> Amolallul Hill

File: QUEST_LV_0200.tsv [hunk 11]
```diff
@@ -2058,49 +2058,49 @@ QUEST_LV_0200_20150317_002057	You can see Commander Danius who looks exhausted.
...
+QUEST_LV_0200_20150317_002102	$The souls are claiming that the Goddess Statue at Amolallul Hills were calling them but, the Sad Owl told you that there isn't a Statue there. Go to Amolallul Hills to find out exactly what happened.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:2102]: Amolallul Hills -> Amolallul Hill (2x)

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**